compiler/src/dmd/iasmgcc.d

```diff
@@ -54,6 +54,14 @@ public Statement gccAsmSemantic(GccAsmStatement s, Scope* sc)
         ptoklist = &(*ptoklist).next;
         *ptoklist = null;
     }
+    // Append closing `;` location.
+    *ptoklist = p.allocateToken();
+    (*ptoklist).value = TOK.semicolon;
+    (*ptoklist).loc = s.loc;
+    ptoklist = &(*ptoklist).next;
+    *ptoklist = null;
```

Comment:
This, so that hitting an unexpected `;` token results in column being reported at:
```
    asm { (; }
           ^
```
And not
```
    asm { (; }
^
```

Reply:
Hmm, I find that rather surprising, seeing that the semicolon's LoC is the asm-statement's?

Reply:
The line was the same thanks to the fixup below around line 65-67